src/applications/disability-benefits/526EZ/sass/disability-benefits.scss

```diff
@@ -82,6 +82,10 @@
     margin-bottom: 1em;
   }
 
+  label {
```

Comment:
This fix is copied in the V1 form so that legacy users do not encounter the same bug